src/iop/sigmoid.c

```diff
@@ -621,29 +758,72 @@ void gui_init(dt_iop_module_t *self)
   gtk_widget_set_tooltip_text(g->hue_preservation_slider, _("optional correction of the hue twist introduced by\n"
                                                             "the per-channel processing method."));
 
-  // collapsible section
+  GtkWidget *main_box = self->widget;
+
+  // primaries collapsible section
+  dt_gui_new_collapsible_section
+    (&g->primaries_section,
+     "plugins/darkroom/sigmoid/expand_primaries",
+     _("primaries"),
```

Comment:
I'll still think of this for a bit. It would be nice to make it clear somehow that this just adjusts the primaries where the per-channel process takes place.